app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -1,8 +1,33 @@
-<p>Hello <%= @resource.email %>!</p>
-
-<p>Someone has requested a link to change your password. You can do this through the link below.</p>
-
-<p><%= link_to 'Change my password', edit_password_url(@resource, reset_password_token: @token) %></p>
-
-<p>If you didn't request this, please ignore this email.</p>
-<p>Your password won't change until you access the link above and create a new one.</p>
+<table class="section">
+  <tr>
+    <td>
+      <table>
+        <tr>
+          <td>
+            <h1 class="section-header">Hello <%= @resource.email %>!</h1>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <h3 class="section-subheader">Someone has requested a link to change your password. You can do this through the link below:</h3>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <%= link_to 'Change my password', edit_password_url(@resource, reset_password_token: @token), class: 'button' %>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p>If you didn't request this, please ignore this email.</p>
+          </td>
+        </tr>
+        <tf>
+          <td>
+            <p>Your password won't change until you access the link above and create a new one.</p>
```

Comment:
Gmail is wrapping some table elements with `<span class="im">` which might be causing the layout issues - http://stackoverflow.com/questions/16689882/gmail-wraps-certain-html-elements-in-a-class-called-im. I'll update the tables accordingly for further testing